apps/condo/domains/user/utils/serverSchema/index.js

```diff
@@ -197,7 +200,34 @@ async function markTokenAsUsed (context, tokenType, tokenAction, sender) {
     }
 }
 
+async function createUserAndSendLoginData ({ context, userData }) {
+    const user = await User.create(context, userData, {
+        errorMapping: {
+            '[password:minLength:User:password]': ERRORS.INVALID_PASSWORD_LENGTH,
+            '[password:rejectCommon:User:password]': ERRORS.PASSWORD_IS_FREQUENTLY_USED,
```

Comment:
Why those errors aren't exists in constants?

Reply:
Idk, i'm just moved this logic from `registerUser` to function to reuse it in `inviteNewOrganizationEmployee`